elevate-mentoring/controllers/mentoring.js

```diff
@@ -31,11 +31,50 @@ const rolePermissions = async (req, res, responses) => {
 	})
 }
 
+const mentorDetails = async (req, res, responses) => {
+	const selectedConfig = routeConfigs.routes.find((obj) => obj.sourceRoute === req.sourceRoute)
+	
+	const parameterisedRoute = req.params.id ? selectedConfig.targetRoute.path.replace('/:id', `/${req.params.id}`) : selectedConfig.targetRoute.path;
+	  let headers
+  
+	  if (req.params.id) {
+		headers = {
+		  'internal_access_token': req.headers['internal_access_token'],
+		  'Content-Type': 'application/json',
+		  'X-auth-token': req.headers['x-auth-token'],
+		}
+	  }
+
+	  console.log("response ============ headers  ",headers);
+	
+	 let response = await requesters.get(req.baseUrl, parameterisedRoute,headers,{})
+	  return res.json(response)
+
+}
+
+const mentoringProfile = async (req, res, responses) => {
```

Comment:
@rakeshSgr check if this is needed, can it be pass through?

Reply:
yes its required.